optaweb-vehicle-routing-backend/pom.xml

```diff
@@ -130,6 +130,11 @@
       <artifactId>spring-boot-configuration-processor</artifactId>
       <optional>true</optional>
     </dependency>
+    <dependency>
+      <groupId>org.optaplanner</groupId>
+      <artifactId>optaplanner-test</artifactId>
```

Comment:
@ge0ffrey This shouldn't be necessary, I think. The dependency should be provided by `optaplanner-spring-boot-starter`.

Reply:
Can we do that and still have it scoped as a test dependency?

Reply:
Or would we rather just add it to the production dependencies for convenience?

Reply:
@triceo ^^

Reply:
Added to this issue: https://issues.redhat.com/browse/PLANNER-1957